ui/react-components/utils/AppointmentListSidePanelWrapper/getFilteredNodesOnToggle.js

```diff
@@ -0,0 +1,30 @@
+const getFilteredNodesOnToggle = (transformedTreeData,filteredCheckedData) => {
+  const filteredNodesList = transformedTreeData
+    .filter(
+      element =>
```

Comment:
Similar cases in this file. IMHO, giving a named function would give me flexibility to know what is filter condition/map login. That way I would be able to understand the code quicker without going through the JS code myself every-time.

Reply:
We have refactor this function also